lxd/storage_volumes.go

```diff
@@ -889,8 +889,8 @@ func doVolumeMigration(s *state.State, r *http.Request, requestProjectName strin
 		return response.InternalError(err)
 	}
 
-	resources := map[string][]string{}
-	resources["storage_volumes"] = []string{fmt.Sprintf("%s/volumes/custom/%s", poolName, req.Name)}
```

Comment:
That's a good question. The reason about this is when the resources are rendered in `func (op *Operation) Render()`, the resource key (here it was previously `storage_volumes`) is used to build an URL. In every parts where we were seeing `resources["storage_volumes"] = ...`, the associated calling endpoint (the swagger one) was in fact starting by `storage-pools`, thus justifying this change. I'm glad you ask because I was not entirely sure about this change ;) @tomponline what do you think ?

Reply:
Apart from potentially the snapshot resources (see my earlier comment) no output should change in this pr.

Reply:
Oh and the inclusion of project param (aa a separate commit)